frontend/lib/swr-cache/index.ts

```diff
@@ -111,6 +111,12 @@ export const withCache = <
          req.write(postData);
          req.end(() => {
             const elapsed = performance.now() - start;
+            logger.info('revalidation request end', {
+               path: req.path,
+               method: req.method,
+               postData,
```

Comment:
It seems like this (`postData`) is the only data we aren't already logging, so I'd vote for just logging that.

The lines below log this:
`cache.findProduct.stale => 12.97ms`  -- This shows *which cache* and *how long it took*